### DISCUSSION TITLE
New formula for wasmtime-c-api or extend existing wasmtime formula?

### DISCUSSION THREAD

**Discussion by eviltwin** [Homebrew/discussions#2248] [Writing Formulae/Casks]
I'm looking at doing some development using wasmtime's c api. It's part of the [same git repository](https://github.com/bytecodealliance/wasmtime/tree/main/crates/c-api) as the main wasmtime package but is built and installed with a separate command as shown [in their docs](https://docs.wasmtime.dev/contributing-building.html#building-the-wasmtime-c-api).

So my question is should I be creating a new formula and submitting that for review or should I extend the existing wasmtime package in some way?

(note that the C API is a completely optional part of wasmtime and, since wasmtime is written in rust, is more akin to something like python bindings for a c library...)

**Comment by carlocab** [comment]
If it's akin to Python bindings for a C library, then it should be part of the same formula and not a separate one. Feel free to open a PR to extend the existing `wasmtime` formula.
